wd_demo.au3

```diff
@@ -128,7 +128,12 @@ Func RunDemo($idDebugging, $idBrowsers)
 		For $iIndex = 0 To UBound($aDemoSuite, $UBOUND_ROWS) - 1
 			If $aDemoSuite[$iIndex][1] Then
 				ConsoleWrite("+Running: " & $aDemoSuite[$iIndex][0] & @CRLF)
-				Call($aDemoSuite[$iIndex][0])
+				Switch $aDemoSuite[$iIndex][0]
+					Case 'DemoPrint'
+						Call($aDemoSuite[$iIndex][0], $idBrowsers)
+					Case Else
+						Call($aDemoSuite[$iIndex][0])
+				EndSwitch
```

Comment:
We should look for a better way to implement this that doesn't involve hardcoding function names. I think we could make this data driven by adding a column to $aDemoSuite. Also, it would make more sense to me to pass the browser name instead of the control ID. This would prevent the need from having code like `$aBrowsers[_GUICtrlComboBox_GetCurSel($idBrowsers)][0]` in multiple routines.

Reply:
[Ok I made new ISSUE](https://github.com/Danp2/WebDriver/issues/155) as this should be in separate PR